airflow/providers/amazon/aws/operators/rds.py

```diff
@@ -59,6 +58,12 @@ def _describe_item(self, item_type: str, item_name: str) -> list:
         elif item_type == 'event_subscription':
             subscriptions = self.hook.conn.describe_event_subscriptions(SubscriptionName=item_name)
             return subscriptions['EventSubscriptionsList']
+        elif item_type == "db_instance":
```

Comment:
These are the same changes as those made to `RdsBaseSensor` in #26003.
Related: https://github.com/apache/airflow/issues/25952#issuecomment-1237585545